nutils/matrix/_scipy.py

```diff
@@ -82,7 +82,8 @@ def mycallback(arg):
                 if callback:
                     callback(res)
                 reformat(100 * numpy.log10(max(atol, res)) / numpy.log10(atol))
-            lhs, status = solverfun(self.core, rhs, M=precon, tol=0., atol=atol, callback=mycallback, **solverargs)
+            solverargs['rtol' if scipy.version.version >= '1.12' else 'tol'] = 0
```

Comment:
It looks like the documented interface for SciPy's version is `scipy.__version__`. See https://github.com/scipy/scipy/blob/main/scipy/__init__.py#L39 . There's no mention of `scipy.version`.

Reply:
Fixed